setup.py

```diff
@@ -1,22 +1,45 @@
-from os import path
+import os
+import subprocess
+import sys
 
 from setuptools import setup
+from setuptools.command.install import install
 
-FULLVERSION = '0.0.1'
+FULLVERSION = '0.0.2-2'
 VERSION = FULLVERSION
 
+class PostInstall(install):
+    """Post install script to install github dependencies."""
+
+    pkgs = "git+https://github.com/GlacioHack/GeoUtils.git"
+
+
+    def run(self):
+        install.run(self)
+
+        print(subprocess.getoutput(f"{sys.executable} -m pip install --force-reinstall {self.pkgs}"))
+
+
+
+with open(os.path.join(os.path.dirname(__file__), "README.md")) as infile:
+    LONG_DESCRIPTION = infile.read()
+
 setup(name='xdem',
       version=FULLVERSION,
-      description='',
-      url='',
+      description='Set of tools to manipulate Digital Elevation Models (DEMs) ',
+      long_description=LONG_DESCRIPTION,
+      long_description_content_type="text/markdown",
+      url='https://github.com/GlacioHack/xdem',
       author='The GlacioHack Team',
+      author_email="john@doe.com",
```

Comment:
is john@doe.com really necessary here? :O

Reply:
Unfortunately yes... I don't remember how to reproduce the error, but I had to add an email address for everything to work!

Reply:
Added a small comment for the future in a7734de